packages/client/src/workflow-client.ts

```diff
@@ -1196,6 +1197,7 @@ export class WorkflowClient extends BaseClient {
   protected async _signalWithStartWorkflowHandler(input: WorkflowSignalWithStartInput): Promise<string> {
     const { identity } = this.options;
     const { options, workflowType, signalName, signalArgs, headers } = input;
+    const jsonConverter = new JsonPayloadConverter();
```

Comment:
Use the data converters configured on the client (i.e. custom payload converter and codec).

Reply:
yup, done